public/javascripts/data.js

```diff
@@ -148,6 +148,23 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
         'Vertical Slider': 'vertical',
         'Picker': 'picker'
     };
+    var mediaTypes = {
+         'Image': 'image/*',
+         'Selfie': 'image/*',
+         'Annotate': 'image/*',
+         'Draw': 'image/*',
+         'Signature': 'image/*',
+         'Audio': 'audio/*',
+         'Video': 'video/*',
+         'Selfie Video': 'video/*'
+    };
+    var mediaAppearances = {
+          'Signature': 'signature',
+          'Annotate': 'annotate',
+          'Draw': 'draw',
+          'Selfie': 'new-front',
+          'Selfie Video': 'new-front'
```

Comment:
new-front is correct for selfie-video. @lognaturel had a copy and paste error in the issue description.